### ISSUE TITLE
config for /27 network not working as expected.

### ISSUE THREAD

**Issue by marktroberts** [nextdns/nextdns#35]
*configs redacted*

host 10.10.10.180 is showing up in the log for conf1 when it should be matching conf2.  No, it is not one of the MAC addresses listed.

config 10.10.10.128/27=<conf1>
config 28:a0:2b:56:e9:66=<conf1>
config 84:89:ad:7c:e3:db=<conf1>
config f4:37:b7:7f:fd:6a=<conf1>
config 70:48:0f:b2:71:c1=<conf1>
config 10.10.10.160/27=<conf2>

**Comment by rs** [comment]
Can you complete your example with the actual conf1/2 you assigned to each config line? It is hard to follow otherwise.

**Comment by marktroberts** [comment]
You can see this host below is 10.10.10.21, so this should fall in config
961783, but its showing up in the logs for config 552462.

config 10.10.10.128/27=552462
config 28:a0:2b:56:e9:66=552462
config 84:89:ad:7c:e3:db=552462
config f4:37:b7:7f:fd:6a=552462
config 70:48:0f:b2:71:c1=552462
config 10.10.10.160/27=9e2d41
config 10.10.10.192/26=d6ee8e
config 10.10.10.0/27=961783
config 10.10.10.32/27=3ea4e8
config 10.10.10.64/27=3ea4e8
config 10.10.10.96/27=869a1f
config 25ead8

[image: image.png]

**Comment by rs** [comment]
I can't reproduce. See the unit test here: 0dd101b

Are you sure the MAC does not match? Can you try removing the MAC based configs?

**Comment by rs** [comment]
Found it
